RELEASE/scripts/autoscend.ash

```diff
@@ -7142,6 +7150,10 @@ boolean L11_unlockEd()
 	if(get_property("auto_tavern") != "finished")
 	{
 		print("Uh oh, didn\'t do the tavern and we are at the pyramid....", "red");
+
+		// Forcing Tavern.
+		set_property("auto_forceTavern", true);
+		if (L3_Tavern()) return true;
```

Comment:
should just return false here rather than calling L3_Tavern()

Reply:
Done.